generated/proto/persist_fs_commitlog.proto

```diff
@@ -21,5 +21,6 @@ message CommitLog {
 
 message CommitLogMetadata {
 	string id = 1;
-	uint32 shard = 2; 
+	string namespace = 2;
+	uint32 shard = 3;
```

Comment:
This is going to be compatibility breaking, should just remember that as we upgrade things =]